edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java

```diff
@@ -194,6 +187,13 @@ private boolean doAction(LogInvocation logInvocation) {
 				logInvocation.setAction(Action.NONE, null);
 				return false;
 			}
+		
+		// process not lower log levels in catch blocks
+		if (logInvocation.getInCatchBlock() && currentLogLevel.intValue() > rejuvenatedLogLevel.intValue()) {
+			this.logInvsNotLoweredInCatch.add(logInvocation);
+			logInvocation.setAction(Action.NONE, null);
+			return false;
+		}
```

Comment:
This setting could be turned on/off in the loop which produces different results. Thus, this setting should be the last to keep the number of log invocations which are accepted by other settings the same.